views/components/promo-card/macro.njk

```diff
@@ -35,7 +35,8 @@ featured: {Boolean} - Use featured style for promo card
                     {% endcall %}
                 </p>
             {% endif %}
-            <h3 class="promo-card__title t4">
+            {% set level = props.headingLevel | default(3, true) %}
```

Comment:
Worth putting a sanity check in here to avoid it being higher than 6? Not that I don't trust us...

Reply:
Good call.
```js
if (commitAuthor = 'davidrapson' && headingLevel > 6) {
  throw new Error('Nope')
}
```